ios/RNGoogleSignin/RNGoogleSignin.m

```diff
@@ -91,8 +91,8 @@ - (instancetype)init {
 RCT_EXPORT_METHOD(signInSilently:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
-  if ([self.promiseWrapper setPromiseWithInProgressCheck:resolve rejecter:reject fromCallSite:@"signInSilently"]) {
```

Comment:
please keep the old code on this line, thanks

Reply:
@vonovak resolved in 99a12b32